quires.sql

```diff
@@ -51,8 +51,7 @@ WHERE date_of_birth BETWEEN DATE '1990-01-01' AND DATE '2000-12-31'
 GROUP BY species;
 
 
-/* Vet clinic database: query multiple tables */
-
+/* Vet-clinic database: query multiple tables */
 
```

Comment:
- [ ] Also, your query to this question - `List all Digimon owned by Jennifer Orwell.` produced the wrong result. Infact, your data is duplicated in three places. 

![image](https://user-images.githubusercontent.com/84156714/230434862-83e41af0-a06a-47d2-8ffb-8b55c2c7e680.png)

Reply:
@bobb-Rob 
thanks for your good and valuable feedback.
your comment is very useful I fixed the issue you can see it now,

![query List all Digimon owned by Jennifer Orwell](https://user-images.githubusercontent.com/108926875/230637894-5efbcca9-c71b-4677-a81f-cb1cf5825ed9.JPG)